docs/source/KGbib.bib

```diff
@@ -169,3 +169,18 @@ @article{TripleRE
   year         = {2022},
   journal      = {arXiv preprint arXiv:2209.08271},
 }
+
+@article{InterHT,
+  author    = {Baoxin Wang and
+               Qingye Meng and
+               Ziyue Wang and
+               Honghong Zhao and
+               Dayong Wu and
+               Wanxiang Che and
+               Shijin Wang and
+               Zhigang Chen and
+               Cong Liu},
+  title     = {InterHT: Knowledge Graph Embeddings by Interaction between Head and Tail Entities},
```

Comment:
I had a quick look at the generated docs, I believe we need to put titles in double brackets here `{{...}}` otherwise capital letters are not displayed corrected :)

Reply:
Thanks for checking so carefully 👍